index.html

```diff
@@ -703,15 +703,17 @@ <h2 class='informative'>Implementation Patterns</h2>
     </section>
 
     <section id="messaging-spi">
-      <h2>Messaging SPI</h2>
+      <h2>Messaging</h2>
       <section class="informative">
+	<h3>Introduction</h3>
         <p>
-          Non-normative: the messaging SPI defines when messages are emitted by a Fedora
+          This section defines when messages are emitted by a Fedora
           implementation, the minimal set of data contained in those messages and how
           those data are serialized. These messages may occur synchronously or asynchronously
 	  with the API operations that cause them to be emitted. They are typically used to
 	  support external integrations. The structure of these messages draws upon the
 	  existing [[activitystreams-core]] and [[ldn]] specifications.
+	  The messaging interface is not defined here and will be implementation specific.
```

Comment:
I do not agree with this sentence. The messaging interface _is_ defined here and it consists of the next three sections. It defines the required data, the serialization format and when events are emitted. What else needs to be defined? Yes, an implementation has some room for interpretation, but by saying that a message must conform to the Activity Streams 2.0 specification, we're being pretty clear.